src/background/isolation.ts

```diff
@@ -23,10 +23,14 @@ export class Isolation {
   private mac!: MultiAccountContainers;
   private management!: Management;
   private utils!: Utils;
+  private intervalIsolationTarget: number;
+  private intervalIsolationInterval: number;
```

Comment:
Similar to the preference wording, maybe something like `reactivateTargetTime` and `reactivateInterval`?

Reply:
Gotcha - Makes sense.  So is this something that'll be better for me to clean up and commit to my branch?  (I'm a bit new to contributions.)

Also-  I just realized that my main brand might be behind yours.  Would you prefer I clean that up?  That might require a new branch, so maybe not worth it.

Reply:
> So is this something that'll be better for me to clean up and commit to my branch? (I'm a bit new to contributions.)

Sure, usually the work is done in the original PR branch. You could just add new commits or change (amend) existing commits. I'll merge only one squashed commit, so there's not much difference to which approach you choose. And if you have further questions about contributing, feel free to ask!

> Also- I just realized that my main brand might be behind yours. Would you prefer I clean that up? That might require a new branch, so maybe not worth it.

You could [sync the PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork), though preferably that's done by [rebasing](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request). However, as long as there's no merge conflict (that's indicated at the bottom of the PR if there's one), there's no need to sync the branch from a merging perspective. Though, if you want to test with the latest changes from the main branch, then you might want to sync.

Reply:
Sticking with the naming convention, using autoEnableTargetTime and autoEnableInterval.